### DISCUSSION TITLE
Does Keda based azure sevice bus trigger function works on AKS Window Docker images ?

### DISCUSSION THREAD

**Discussion by atulthummar** [kedacore/keda#2095] [Q&A / Need Help]
Does Keda based azure sevice bus trigger function works on AKS Window Docker images ?

**Comment by arschles** [comment]
@atulthummar all of the scalers should work on any `Deployment` in AKS (or any other Kubernetes distro), so if you can run Windows images in your cluster, you can scale them with KEDA, including with its service bus trigger.

**Reply by tomkerkhove** [reply]
As long as the KEDA control plane runs on Linux nodes

**Comment by atulthummar** [comment]
**C:\WINDOWS\system32>kubectl logs jobprocessor-function-cfb955886-fmqbd -p**
  It was not possible to find any installed .NET Core SDKs
  Did you mean to run .NET Core SDK commands? Install a .NET Core SDK from:
      https://aka.ms/dotnet-download

getting above while checking pod log and pod status CrashLoopBackOff

**Reply by atulthummar** [reply]
C:\WINDOWS\system32>kubectl logs jobprocessor-function-5c57b6cb95-4hk5n
Hosting environment: Production
Content root path: C:\app
Now listening on: http://[::]:80
Application started. Press Ctrl+C to shut down.

**Reply by tomkerkhove** [reply]
This is an issue related to your Azure Function and not KEDA, sorry.

We just scale your app but don't influence the runtime.
